packages/global/templates/content/print.marko

```diff
@@ -100,52 +102,84 @@ $ const displayPublishedDate = ["event", "webinar", "contact", "company"].includ
             block-name=cardBlock
             obj=content
           />
-
-          <if(primaryImage.isLogo)>
-            <marko-web-page-image
-              obj=primaryImage
-              fluid=false
-              width=250
-            />
-          </if>
-
-          <if(type !== "contact")>
-            <default-theme-content-attribution obj=content />
-            <default-theme-page-dates|{ blockName }|>
-              <if(type === "event")>
-                <marko-web-content-start-date block-name=blockName obj=content />
-                <marko-web-content-end-date block-name=blockName obj=content />
+          $ const accessLevels = getAsArray(content, "userRegistration.accessLevels");
+          <marko-web-identity-x-access|context|
+            enabled=requiresRegistration
+            required-access-level-ids=accessLevels
+          >
+            <if(!context.canAccess || context.requiresUserInput)>
+              $ const body = getContentPreview({ body: content.body, selector: "p:nth-of-type(1)" });
```

Comment:
@B77Mills - Can you update the selector to use the new logic?  Instead of `p:nth-of-type(1)`, I believe it is `p:lt(3)`.  CSS _may_ need to be updated; not positive.